### DISCUSSION TITLE
Un-mod Taskbar Running and Active Indicators

### DISCUSSION THREAD

**Discussion by ApolloAzrael** [ramensoftware/windhawk-mods#832] [Q&A]
I'm currently attempting to use the RosePine theme for my taskbar with a few adjustments, namely using large program icons. I really don't like the look of the active and running indicators, as they barely poke out from beneath the icon most times. I tried to remove the target from the active control styles, but I'm still seeing them. Any advice on how to get rid of them?

**Comment by m417z** [comment]
Hi, taskbar themes are usually discussed here:
https://github.com/ramensoftware/windows-11-taskbar-styling-guide

Perhaps the author of the theme, @lunar-os, can help.

**Comment by lunar-os** [comment]
> I'm currently attempting to use the RosePine theme for my taskbar with a few adjustments, namely using large program icons. I really don't like the look of the active and running indicators, as they barely poke out from beneath the icon most times. I tried to remove the target from the active control styles, but I'm still seeing them. Any advice on how to get rid of them?

it should be near the top of the targets i have set. it's one with a lot of styles mostly with the same colour(the rose colour). sorry i'm not at my computer so i cant find the exact name
